scripts/integration-tests/prometheus/docker-compose.yml

```diff
@@ -38,5 +38,13 @@ services:
     image: prom/prometheus:latest
     volumes:
       - "./:/etc/prometheus/"
+  grafana:
```

Comment:
Just curious if grafana used within the test?

Reply:
It is not, we're going to be using this as a hack to run this locally heh.